templates/dashboard/subscription.php

```diff
@@ -7,69 +135,96 @@
             <strong><?php esc_html_e( 'Package & billing details: ', 'wp-user-frontend' ); ?></strong>
             <?php echo esc_html( $billing_amount . ' ' . $recurring_des ); ?>
         </div>
-        <?php  if ( is_wp_error( $user_sub ) ) { ?>
+        <?php if ( is_wp_error( $user_sub ) ) { ?>
         <div>
             <strong><?php esc_html_e( 'Subscription Status: ', 'wp-user-frontend' ); ?></strong>
             <?php esc_html_e( 'Subscription Expired!', 'wp-user-frontend' ); ?>
         </div>
-        <?php } else {
-             if ( ! empty( $user_sub['total_feature_item'] ) && -1 === $user_sub['total_feature_item'] ) { ?>
+			<?php
+        } else {
+			if ( ! empty( $user_sub['total_feature_item'] ) && -1 === $user_sub['total_feature_item'] ) {
+				?>
                 <div><strong><?php esc_html_e( 'Number of featured item: ', 'wp-user-frontend' ); ?></strong><?php echo esc_html( $user_sub['total_feature_item'] ); ?></div>
             <?php } ?>
             <div>
                 <strong><?php esc_html_e( 'Remaining post: ', 'wp-user-frontend' ); ?></strong>
                 <?php
                 $i = 0;
+                $post_count = 0;
                 if ( ! empty( $user_sub['posts'] ) ) {
                     foreach ( $user_sub['posts'] as $key => $value ) {
                         $value = intval( $value );
-
                         if ( 0 === $value || -1 === $value ) {
                             continue;
                         }
-
                         $post_type_obj = get_post_type_object( $key );
-
                         if ( ! $post_type_obj ) {
                             continue;
                         }
-                        $value = ( $value == '-1' ) ? __( 'Unlimited', 'wp-user-frontend' ) : $value; ?>
-                        <div><?php echo esc_html( $post_type_obj->labels->name ) . ': ' . esc_html( $value ); ?></div>
+                        ++$post_count;
+                    }
+                }
+                ?>
+                <div id="wpuf-remaining-posts-list">
+                <?php
+                $i = 0;
+                if ( ! empty( $user_sub['posts'] ) ) {
+                    foreach ( $user_sub['posts'] as $key => $value ) {
+                        $value = intval( $value );
+                        if ( 0 === $value || '-1' === $value ) {
+                            continue;
+                        }
+                        $post_type_obj = get_post_type_object( $key );
+                        if ( ! $post_type_obj ) {
+                            continue;
+                        }
+                        $value = ( '-1' === $value ) ? __( 'Unlimited', 'wp-user-frontend' ) : $value;
```

Comment:
on `line: 174` we are already jumping out the iteration by checking if the $value is '-1'. So it won't come to `line: 181` I believe